ssl/quic/quic_lcidm.c

```diff
@@ -554,3 +584,16 @@ int ossl_quic_lcidm_debug_add(QUIC_LCIDM *lcidm, void *opaque,
     lcid_obj->type      = LCID_TYPE_NCID;
     return 1;
 }
+
+int ossl_quic_lcidm_get_unused_cid(QUIC_LCIDM *lcidm, QUIC_CONN_ID *cid)
+{
+    int i;
+
+    for (i = 0; i < 10; i++) {
+        if (lcidm_generate_cid(lcidm, cid) &&
+            (lcidm_get0_lcid(lcidm, cid) == NULL))
```

Comment:
useless parenthesis pair here

Reply:
I think the added parenthesis makes this more understandable

Reply:
This is against our coding style and IMO putting `&&` on the beginning of the second line (which needs to be done according to our coding style) makes this understandable very well without them.